src/drivers/distance_sensor/lightware_sf45_serial/lightware_sf45_serial.cpp

```diff
@@ -633,22 +708,22 @@ void SF45LaserSerial::sf45_send(uint8_t msg_id, bool write, int *data, uint8_t d
 	packet_len = (uint8_t)len;
 
 	// DEBUG
-	for (uint8_t i = 0; i < packet_len; ++i) {
+	/*for (uint8_t i = 0; i < packet_len; ++i) {
 		PX4_INFO("INFO: Send byte: %d", packet_buff[i]);
-	}
+	}*/
```

Comment:
Any reason why you commented out the DEBUG code? It could be quite helpful when having issues in the future. You can change it from PX4_INFO to PX4_DEBUG to avoid it spamming in "normal" usage modes.

Reply:
Yeah, this is only useful when initializing the sensor (the driver sends data to the sensor to init) so there is probably no harm in setting to debug